start-flask.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+# load configs if present
+if [ "$FLASK_ENV" != "development" ]; then
+  if [ -f /run/secrets/environment ]
+  then
+      source /run/secrets/environment
```

Comment:
Preferably you should configure Kubernetes to [load the secrets directly into the environment](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables), rather than sourcing a file on start. (I've deployed some of our older applications with an `environment` file because that's how they were already set up, but for anything new we should switch away from doing it that way.)

Either way, the secret will need to be set up in `kubernetes/deployment.tmpl`.

Reply:
Awesome, thank you!

Reply:
Can you take a look at this and see how I've set it up? I think this is correct but I'm not completely sure.